notifications_utils/broadcast_areas/__init__.py

```diff
@@ -147,6 +147,10 @@ def get_all_areas(self):
                 yield area
 
     def get_areas(self, *area_ids):
+        # allow people to call `get_areas('a', 'b') or get_areas(['a', 'b'])`
```

Comment:
🤨

Reply:
i'd rather just the second, but in the sake of keeping it non-breaking